lib/inih/ini.c

```diff
@@ -51,6 +51,18 @@ static char* lskip(const char* s)
     return (char*)s;
 }
 
+/* Strip quotes from single and double quoted strings */
+static char* unquote(char* s)
+{
+    int l = strlen(s);
+    if (s[0] == s[l] && (s[0] == '\'' || s[0] == '"'))
```

Comment:
```suggestion
    if (s[0] == s[l - 1] && (s[0] == '\'' || s[0] == '"'))
```

In review, I think this should be `s[l - 1]` versus `s[l]`.

Reply:
Empty case should be handled as well.

Reply:
Empty case refers to `''` and `""`?
How should the empty case be handled?
Currently this will make `s` point to `\0` (really, `\0\0`); an empty c-string.